src/directory/directory.js

```diff
@@ -1,1114 +1,1114 @@
 const directory = {
   lib: {
     productRoot: {
-      title: "Amplify Libraries",
-      route: "/lib",
+      title: 'Amplify Libraries',
```

Comment:
Was this file not up-to-date with prettier settings? 👀

Reply:
Yeah, as soon as I saved it reformatted it, to single quotes.